README.md

```diff
@@ -34,3 +34,9 @@ This is the first release based on the new artifact ids (which now include the s
 
 At this stage support for the M1 and M2 APIs is relatively complete. Support for annotations may need refinements. Some constraints may not yet been verified.
 Import and export from and to EMF is a work in progress.
+
+### Version 0.2.1
```

Comment:
I think it could make sense to have a separate Changelog file for this

Reply:
Ha, I thought we decided to not have separate changelog files. Also: the `##` heading says "Changelog".

Reply:
Sorry, my bad

Reply:
No worries :D